src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildANetCoreAppWithWap.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Xunit.Abstractions;
+
+namespace Microsoft.NET.Publish.Tests
+{
+    public class GivenThatWeWantToBuildANetCoreAppWithWap : SdkTest
+    {
+        public GivenThatWeWantToBuildANetCoreAppWithWap(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        [RequiresSpecificFrameworkFact("net5.0")]
```

Comment:
I think this can be a simple `Fact`, was that not working?

```suggestion
        [Fact]
```

Reply:
Since the WPF project requires .NET5.0, I assumed I would need this attribute.  I'll try it without.